pytest_fixtures/component/rh_cloud.py

```diff
@@ -4,10 +4,10 @@
 
 
 @pytest.fixture(scope='module')
-def rhcloud_manifest_org(module_target_sat, module_extra_rhel_entitlement_manifest):
```

Comment:
@ColeHiggins2 should we cleanup the code and remove `module_extra_rhel_entitlement_manifest` fixture as it was only used by rhc tests? wdyt?

Reply:
also `module_extra_rhel_entitlement_manifest_org`

Reply:
Yes, we should remove all of the `extra_rhel` fixtures for stream/6.16. There is no benefit to using more than 1 of each subscription on an SCA manifest. If the affected tests pass with SCA on earlier Satellite versions, we can cherrypick that change back as well.

Reply:
I have another jira ticket that cleans up all fixtures and code around sca. so I will do it then